generatebundlefile/promote.go

```diff
@@ -151,17 +151,30 @@ func (c *SDKClients) PromoteHelmChart(repository, authFile string, crossAccount
 		return fmt.Errorf("Helm chart doesn't have requires.yaml inside %s", err)
 	}
 	// Unpack requires.yaml into a GO struct
-	helmRequires, err := validateHelmRequires(f)
+	helmRequiresImages, err := validateHelmRequires(f)
 	if err != nil {
 		return fmt.Errorf("Unable to parse requires.yaml file to Go Struct %s", err)
 	}
-	// Add the helm chart to the struct before looping through lookup/promote since we need it promoted too.
-	helmRequires.Spec.Images = append(helmRequires.Spec.Images, Image{Repository: chartName, Tag: version, Digest: sha})
+
+	// Create a 2nd struct since the the helm chart is going to Public ECR while the image are going to Private ECR.
+	// Remove this once we add support for Private ECR pull in the Helm Driver.
```

Comment:
I think you can get rid of this comment as long term when kubelet authentication is used we will need the helm charts in public ECR.